proxy/nginx.conf.template

```diff
@@ -296,6 +296,10 @@ server {
     proxy_set_header X-Forwarded-Proto "https";
 
     proxy_pass $authUrl;
+
+    proxy_buffer_size 128k;
```

Comment:
Does it _really_ need 128 kB for headers? How huge is that stuff?

Reply:
Decreased and lets test if it works with it.